src/Playwright.MSTest/Playwright.MSTest.csproj

```diff
@@ -9,7 +9,7 @@
       and fixtures to enable using it within MSTest.
     </Description>
     <PackageIcon>icon.png</PackageIcon>
-    <TargetFrameworks>netcoreapp3.1;net5.0;net48</TargetFrameworks>
```

Comment:
All of these targets are in netstandard2.0 and the project is a library (and not an executable), so netstandard2.0 is better.